koku/masu/database/report_manifest_db_accessor.py

```diff
@@ -307,7 +307,7 @@ def update_and_get_day_file(self, day, manifest_id):
             counter = manifest.report_tracker[day]
             manifest.report_tracker[day] = counter + 1
             manifest.save(update_fields=["report_tracker"])
-            return f"{day}_{counter}.csv"
+            return f"{day}_manifestid-{manifest_id}_{counter}.csv"
```

Comment:
I assume this means we are going to have to delete data (manually) because the naming format is changing, correct?

Reply:
No, since we are deleting the files now anyway :)